### PR TITLE
docs(READEME.md): change image size

### PR DESCRIPTION
**Pull request by zaunist**
### Type of change:

- [ ] Bugfix
- [ ] New feature provided
- [x] Improve performance
- [ ] Backport patches

### What this PR does / why we need it:

The [image](https://github.com/apache/apisix-ingress-controller#internal-architecture) is to flat, so I change the changed the ratio of height and width.

solved #796 

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix-ingress-controller#community) first**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ This project is currently general availability.
 
 ## Internal Architecture
 
-<img src="./docs/assets/images/apisix-ingress-controller-arch.png" alt="module" width="600" height="313" />
+<img src="./docs/assets/images/apisix-ingress-controller-arch.png" alt="module" width="74.3%" height="55.9%" />
```
Comment (tokers) [inline README.md:58]: May I know why you choose such a ratio?

Comment (zaunist) [reply README.md:58]: Refered this [file](https://github.com/apache/apisix-ingress-controller/pull/301/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R32), and I think a scaled image is more appropriate than a fixed height and width.

**Comment by zaunist** [comment]
cc @gxthrj

**Review by gxthrj — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (tokers) [reply README.md:58]: Got it.

**Review by tokers — APPROVED** [review]

**Pull request merged**